### PR TITLE
Azure getting started refinement

### PR DESCRIPTION
**Pull request by coderbyheart**

### CODE REVIEW

File: docs/azure/GettingStarted/Deploy.rst [hunk 1]
```diff
@@ -87,21 +70,36 @@ To install the nRF Asset Tracker into your Azure account, complete the following
...
+   In this example, ``nrfassettrackerusers`` is used as the initial domain name.
+
+   .. code-block:: bash
 
-   a. Follow the instructions in the `tutorial for registering a web application in Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=applications>`_ and create a B2C tenant and an application. Use ``http://localhost:3000/`` (for local development) and ``https://<your APP_NAME>app.z16.web.core.windows.net/`` as the redirect URLs.
+      # add to .envrc
+      export B2C_TENANT="nrfassettrackerusers"
 
-   #. Enable the implicit grant and hybrid flows for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
+#. After creating the Active Directory B2C, under :guilabel:`User flows`, create the user flow for sign up and sign in (recommended version), and make sure to name the userflow as ``B2C_1_signup_signin``.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
#. Select :guilabel:`User flows`, create the user flow for sign up and sign in (recommended version), and make sure to name the userflow as ``B2C_1_signup_signin``.
```
Should we also mention where to find the User flows? Is it under Policies?

File: docs/azure/GettingStarted/Deploy.rst [hunk 2]
```diff
@@ -87,21 +70,36 @@ To install the nRF Asset Tracker into your Azure account, complete the following
...
 
-#. Create the user flow for sign-up, sign-in, and make sure to name the userflow as ``B2C_1_signup_signin``.
+#. Follow the instructions in the `tutorial for registering a web application in Azure Active Directory B2C <https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=app-reg-ga#register-a-web-application>`_ and register a web application. Use ``https://<your APP_NAME>app.z16.web.core.windows.net/`` as the redirect URL.
 
-   a. Click :guilabel:`Application claims`, select :guilabel:`Show more ...` and then mark :guilabel:`Email Addresses` as a return claim.
+#. After registering the application, under :guilabel:`Authentication`, enable the implicit grant and hybrid flows for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
#. Select :guilabel:`Authentication`, enable the implicit grant and hybrid flows for :guilabel:`Access tokens` and :guilabel:`ID tokens` and click :guilabel:`Save`.
```
Where is the Authentication exactly?

File: docs/azure/GettingStarted/Deploy.rst [hunk 3]
```diff
@@ -87,21 +70,36 @@ To install the nRF Asset Tracker into your Azure account, complete the following
...
+
+      # add to .envrc
+      export APP_REG_CLIENT_ID=...
+
+#. Grant the app registration directory API permissions for the function app:
+
+   a. Click :guilabel:`Expose an API`.
    
-   a. Click :guilabel:`Expose an API` and  set the ``Application ID URI`` field to ``api``.
+   #. Set the ``Application ID URI`` field to ``api``. Click :guilabel:`Save and continue`.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
   #. Set the ``Application ID URI`` field to ``api`` and click :guilabel:`Save and continue`. 
```

File: docs/azure/GettingStarted/Deploy.rst [hunk 4]
```diff
@@ -164,12 +138,16 @@ To install the nRF Asset Tracker into your Azure account, complete the following
...
       :class: highlight
 
-      az monitor activity-log list --correlation-id "*tracking id*" \
+      az monitor activity-log list --correlation-id "*tracking ID*" \\
          | jq '.[].properties.statusMessage | fromjson'
 
    It can take a few minutes for the detailed log message to be populated.
+
+   If the error message does not include a tracking ID navigate to the resource group in the Azure portal and review the deployments.
+   There should be a failed deployment called ``initial-setup``.
```
Comment (peknis) [inline docs/azure/GettingStarted/Deploy.rst]:
```suggestion
   If the error message does not include a tracking ID, navigate to the resource group in the Azure portal and review the deployments.
   There is a failed deployment called ``initial-setup``.
```

File: docs/azure/Upgrading.rst [hunk 5]
```diff
@@ -45,12 +45,16 @@ To verify the validity of a template, use the following command:
...
+    :class: highlight
 
-    az monitor activity-log list --correlation-id "*tracking id*" \
+    az monitor activity-log list --correlation-id "*tracking ID*" \\
         | jq '.[].properties.statusMessage | fromjson'
 
 It can take a few minutes for the detailed log message to be populated.
+
+If the error message does not include a tracking ID navigate to the resource group in the Azure portal and review the deployments.
+There should be a failed deployment called ``initial-setup``.
```
Comment (peknis) [inline docs/azure/Upgrading.rst]:
```suggestion
If the error message does not include a tracking ID, navigate to the resource group in the Azure portal and review the deployments.
There is a failed deployment called ``initial-setup``.
```

File: docs/firmware/azure/BuildingUsingGitHub.rst [hunk 6]
```diff
@@ -11,11 +11,12 @@ After you have forked the repository, `enable GitHub Actions <https://help.githu
 Configuration for firmware connecting to the nRF Asset Tracker for Azure
 ========================================================================
 
-Navigate to the settings of the repository and configure two new Secrets ``AZURE_IOT_HUB_DPS_HOSTNAME``, and ``AZURE_IOT_HUB_DPS_ID_SCOPE`` and assign the DPS host name and DPS ID scope of your Azure IoT Hub MQTT broker to the secrets.
-You can retrieve the values for the secrets by using the following command:
+Navigate to the settings of the repository and configure the secret ``AZURE_IOT_HUB_DPS_ID_SCOPE`` and assign the DPS ID scope of your Azure IoT Hub Device Provisioning to it.
 
-.. code-block:: bash
+You can retrieve the value for the Azure IoT DPS ID scope secret using the following command:
```
Comment (peknis) [inline docs/firmware/azure/BuildingUsingGitHub.rst]:
```suggestion
To retrieve the value for the Azure IoT DPS ID scope secret, use the following command:
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: docs/azure/GettingStarted/Deploy.rst (hunk 1 above)
Comment (coderbyheart) [reply docs/azure/GettingStarted/Deploy.rst]: It's on the left-hand side visible at the top level.
image

File: docs/azure/GettingStarted/Deploy.rst (hunk 2 above)
Comment (coderbyheart) [reply docs/azure/GettingStarted/Deploy.rst]: image

**Review by peknis — APPROVED** [review]

File: docs/azure/GettingStarted/FirmwareConfiguration.rst [hunk 7]
```diff
@@ -5,7 +5,17 @@ Configure the firmware
 
 To configure the :ref:`firmware <firmware-azure-index>`, complete the following mandatory steps:
 
-1. // FIXME: Implement
+1. Run the following command to print the Azure IoT DPS ID scope to which your devices use during device provisioning:
```
Comment (umapraseeda) [inline docs/azure/GettingStarted/FirmwareConfiguration.rst]: Is it 'register'? Or is it :  Run the following command to print the Azure IoT DPS ID scope, which your devices use during device provisioning:
```suggestion
1. Run the following command to print the Azure IoT DPS ID scope to which your devices register during device provisioning:
```

Comment (coderbyheart) [reply docs/azure/GettingStarted/FirmwareConfiguration.rst]: No, the ID scope is used to identify the DSP instance. It is the same for all devices.

Comment (coderbyheart) [reply docs/azure/GettingStarted/FirmwareConfiguration.rst]: https://docs.microsoft.com/en-us/azure/iot-dps/concepts-service#id-scope

Comment (umapraseeda) [inline docs/azure/GettingStarted/FirmwareConfiguration.rst]:
```suggestion
1. Run the following command to print the Azure IoT DPS ID scope, which your devices use during device provisioning:
```

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**